Assets/Scripts/WorldEngine/Manager.cs

```diff
@@ -285,93 +282,32 @@ public class Debug_TracingData
 
     public XmlAttributeOverrides AttributeOverrides { get; private set; }
 
-    public static bool PerformingAsyncTask
-    {
-        get
-        {
-            return _manager._performingAsyncTask;
-        }
-    }
+    public static bool PerformingAsyncTask => _manager._performingAsyncTask;
```

Comment:
Another thing I didn't know was possible. nice! BTW. What IDE/editor are you using?

Reply:
I'm using visual studio 2019